From 2aff5594967b50441ea5313b49c439bfb55b5955 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Thu, 26 Mar 2015 15:14:46 +0800
Subject: [PATCH 0901/1594] MLK-10479 ARM: dts: i.MX6QP: add enet pin setting
 for i.MX6QP

commit cca23dd5005d0c48256562c5561eb6036979b904 from
git://git.freescale.com/imx/linux-2.6-imx.git

Enet has overrun by using the i.MX6Q PAD setting when VDDIO is 1.8V.
Change the DSE from 40Ohm to 80Ohm, the overrun disappear. After measure
the waveform, the tx amplitude reach at 2.4V with DSE 40Ohm, at 1.8V with
DSE 80Ohm. So the patch just set the suitable pad setting for i.MX6QP with
ethernet daughter board.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/boot/dts/imx6qp-sabreauto.dts |   24 ++++++++++++++++++++++++
 1 files changed, 24 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qp-sabreauto.dts b/arch/arm/boot/dts/imx6qp-sabreauto.dts
index 9aa2b9b..d8ac17d 100644
--- a/arch/arm/boot/dts/imx6qp-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6qp-sabreauto.dts
@@ -26,6 +26,30 @@
 	};
 };
 
+&iomuxc {
+	imx6qdl-sabreauto {
+		pinctrl_enet: enetgrp {
+			fsl,pins = <
+				MX6QDL_PAD_KEY_COL1__ENET_MDIO          0x1b0b0
+				MX6QDL_PAD_KEY_COL2__ENET_MDC           0x1b0b0
+				MX6QDL_PAD_RGMII_TXC__RGMII_TXC         0x1b018
+				MX6QDL_PAD_RGMII_TD0__RGMII_TD0         0x1b018
+				MX6QDL_PAD_RGMII_TD1__RGMII_TD1         0x1b018
+				MX6QDL_PAD_RGMII_TD2__RGMII_TD2         0x1b018
+				MX6QDL_PAD_RGMII_TD3__RGMII_TD3         0x1b018
+				MX6QDL_PAD_RGMII_TX_CTL__RGMII_TX_CTL   0x1b018
+				MX6QDL_PAD_RGMII_RXC__RGMII_RXC         0x1b018
+				MX6QDL_PAD_RGMII_RD0__RGMII_RD0         0x1b018
+				MX6QDL_PAD_RGMII_RD1__RGMII_RD1         0x1b018
+				MX6QDL_PAD_RGMII_RD2__RGMII_RD2         0x1b018
+				MX6QDL_PAD_RGMII_RD3__RGMII_RD3         0x1b018
+				MX6QDL_PAD_RGMII_RX_CTL__RGMII_RX_CTL   0x1b018
+				MX6QDL_PAD_GPIO_16__ENET_REF_CLK        0x4001b0a8
+			>;
+		};
+	};
+};
+
 &vgen3_reg {
 	regulator-always-on;
 };
-- 
1.7.5.4

